packages/access-token/src/json-store.js

```diff
@@ -2,6 +2,7 @@
 import fs from 'fs';
 import path from 'path';
 import process from 'process';
+import lockfile from 'proper-lockfile';
```

Comment:
I think this is where the synthetic TS error you're seeing comes from. The [types](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/proper-lockfile/index.d.ts) are stricter than the actual [implementation](https://github.com/moxystudio/node-proper-lockfile/blob/master/index.js).


```suggestion
import { lock } from 'proper-lockfile';
```

or 

```suggestion
import * as lockfile from 'proper-lockfile';
```

Reply:
There are synthetic TS errors for every bare specifier `import` in this file, so I see no reason to try working around this particular instance.